README.md

```diff
@@ -1,5 +1,7 @@
 #  resin-am571x-evm repository
 
+## This BSP has been discontinued and has no further updates planned from Resin.io.
```

Comment:
@willnewton something like:
"This BSP has been discontinued, which means resin.io and resinos.io will no longer offer prebuilt versions for this BSP."